button.go

```diff
@@ -0,0 +1,40 @@
+package engo
+
+// Button is an input which can be either JustPressed, JustReleased or Down. Common uses would be for, a jump key or an action key.
+type Button struct {
+	Triggers []Key
+	Name     string
+}
+
+func (b Button) JustPressed() bool {
```

Comment:
They are but It's still nice for both godoc and the function docs that show up on editors.